core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -117,6 +119,28 @@ private[ui] class ExecutorsPage(
     val maximumMemory = info.maxMemory
     val memoryUsed = info.memoryUsed
     val diskUsed = info.diskUsed
+
+    // Determine Color Opacity from 0.5-1
+    var activeTasksAlpha = 1.0
+    var failedTasksAlpha = 1.0
+    var completedTasksAlpha = 1.0
+    var totalDurationAlpha = 1.0
+    if (info.totalCores > 0) {
+      // activeTasks range from 0 to all cores
+      activeTasksAlpha = (info.activeTasks.toDouble / info.totalCores) * 0.5 + 0.5
+    }
+    if (info.totalTasks > 0) {
```

Comment:
I wanted to do that but the style guide wasn't clear on whether it was allow, I'll update it since it's cleaner. And I'll add the 1.0 cap, I left it off originally because the code was getting busy and hard to read originally.